tests/conftest.py

```diff
@@ -66,6 +67,14 @@ def pytest_addoption(parser):
         help="use firecracker/jailer binaries from this directory instead of compiling from source",
     )
 
+    parser.addoption(
```

Comment:
Nit: It would be cool to have a very simple example somewhere that would help figure out escaping.

Reply:
ok, ignore this, I can see it in the PR description :)